org/mozilla/jss/ssl/SSLVersion.java

```diff
@@ -36,14 +53,24 @@ public static SSLVersion valueOf(int value) {
         throw new IllegalArgumentException("Invalid SSLVersion value: " + value);
     }
 
+    public boolean matchesAlias(String value) {
+        for (String alias : aliases) {
+            if (value.toUpperCase() == alias.toUpperCase()) {
```

Comment:
I think this should be:
```
if (alias.equalsIgnoreCase(value)) {
```
Notes:
* String comparison in Java should be done with equals...() instead of ==.
* The equals...() is called against alias since it's guaranteed not to be null.

Reply:
Ah duh, sorry. I've been not-Java-ing for too long.